src/js/media-poster-image.js

```diff
@@ -15,24 +15,36 @@ template.innerHTML = `
       box-sizing: border-box;
     }
 
+    #container {
+      background-repeat: no-repeat;
+      background-position: var(--media-background-position, var(--media-object-position, center));
+      background-size: var(--media-background-size, var(--media-object-fit, contain));
+    }
     img {
       max-width: 100%;
       max-height: 100%;
       min-width: 100%;
       min-height: 100%;
-      background-repeat: no-repeat;
-      background-position: var(--media-background-position, var(--media-object-position, center));
-      background-size: var(--media-background-size, var(--media-object-fit, contain));
       object-fit: var(--media-object-fit, contain);
       object-position: var(--media-object-position, center);
+      opacity: 1;
+    }
+    img.fade {
+      opacity: 0;
+      transition: opacity 0.15s ease;
+    }
+    img.fade.loaded {
+      opacity: 1;
     }
 
     :host([${MediaUIAttributes.MEDIA_HAS_PLAYED}]) img {
       display: none;
     }
   </style>
 
-  <img aria-hidden="true" id="image"/>
+  <div aria-hidden="true" id="container">
```

Comment:
Potentially, the host itself i.e. `this` below, can be used for the background image so we may not need this new container